test/e2e/framework/syncer.go

```diff
@@ -127,19 +130,44 @@ func WithDownstreamPreparation(prepare func(config *rest.Config, isFakePCluster
 	}
 }
 
-// Start starts a new syncer against the given upstream kcp workspace. Whether the syncer run
-// in-process or deployed on a pcluster will depend whether --pcluster-kubeconfig and
-// --syncer-image are supplied to the test invocation.
-func (sf *syncerFixture) Start(t *testing.T) *StartedSyncerFixture {
+// CreateAndStart creates SyncTarget resource, applies it in the physical cluster,
+// and then starts a new syncer against the given upstream kcp workspace.
+// Whether the syncer runs in-process or deployed on a pcluster will depend
+// whether --pcluster-kubeconfig and --syncer-image are supplied to the test invocation.
+func (sf *syncerFixture) CreateAndStart(t *testing.T) *StartedSyncerFixture {
```

Comment:
am confused. How is a SyncTarget applied to the physical cluster?

Reply:
This looks a whole lot like `Create(t).StartSyncer(t)`, i.e. it overlaps with `Create`, no?

Reply:
Yeah, bad wording. The SyncTarget is **not** what is applied. But the Syncer-related resources are applied in the physical cluster.

Reply:
Fixed in commit 13a4a065c80f0bf3815d69defc0ffcd0728fce2d

Reply:
> This looks a whole lot like Create(t).StartSyncer(t), i.e. it overlaps with Create, no?

Reply:
I changed it to have either:

```
syncerFixture.Create(t).StartSyncer(t)
```
or
```
syncerFixture.Create(t).StartAPIImporter(t).StartHeartBeat(t)
```